core/lib/spree/event/subscriber.rb

```diff
@@ -66,9 +73,11 @@ def event_action(method_name, event_name: nil)
       def subscribe!
         unsubscribe!
         event_actions.each do |event_action, event_name|
-          send "#{event_action}_handler=", Spree::Event.subscribe(event_name) { |event|
-            send event_action, event
-          }
+          subscription = Spree::Event.subscribe(event_name) { |event| send event_action, event }
+          # old mappings, to be removed when Solidus 2.10 is not supported anymore:
+          send "#{event_action}_handler=", subscription
+          # new mappings:
```

Comment:
I think we can get rid of this comment, probably improving a little bit the one with the deprecated code to explain that this other way is the new way of doing things.